docs/automerge_synopsis.md

```diff
@@ -0,0 +1,128 @@
+## Automerge Synopsis
+
+[Join the Automerge Slack community](https://communityinviter.com/apps/automerge/automerge)
+
+---
+### Automerge document pages
+* **Automerge synopsis**
+* [Changelog](https://github.com/restarian/automerge/blob/brace_document/docs/changelog.md)
```

Comment:
These should change to reference `automerge/automerge/blob/master` (or use a relative path).

Reply:
That is done automatically with the *brace_document* command. 
Use **npm run make_docs -- -v** or **brace_document -v -i docs --sort depth -r --navlink** at the project root to make the changes for any particular repo/branch.

Reply:
Also, there is a lot of info at the github repository {Brace document](https://github.com/restarian/brace_document.git)